reflex/testing.py

```diff
@@ -155,6 +155,8 @@ def _initialize_app(self):
                 )
                 self.app_module_path.write_text(source_code)
         with chdir(self.app_path):
+            # ensure config is reloaded when testing different app
+            reflex.config.get_config(reload=True)
```

Comment:
I wonder if we should have a way to specify the config, so multiple configs can exist together and we can avoid the forced re-import

Reply:
we definitely should. i think that can be handled during our upcoming config refactor